third_party/2and3/mypy_extensions.pyi

```diff
@@ -9,3 +9,5 @@ def TypedDict(typename: str, fields: Dict[str, Type[_T]]) -> Type[dict]: ...
 # This type is equivalent to the None type, but the no-op Union is necessary to
 # distinguish the None type from the None value.
 NoReturn = Union[None]
+
+def decorated_type(t: Type) -> Callable[[_T], _T]: pass
```

Comment:
Extremely minor, but I prefer `Type[Any]` over just `Type` to be explicit; typeshed style is to use `...` instead of `pass`, and please add a closing newline.